documentation/docs/configuration/configuration.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 1
+---
+
+# Configuration
+
+The Dynatrace MONitoring As Code tool defines a structure on what files
```

Comment:
nitpick: if you purposely capitalize for monaco add the shorthand afterwards, which also allows you to use the monaco from then on

```
The Dynatrace MONitoring As Code tool (monaco) defines....
```